tests/test_image_utils.py

```diff
@@ -1,24 +1,27 @@
+import unittest
+
 from src import image_utils
 import numpy as np
 
-""" Varon Ratio Tests """
-S1 = np.array([[0.1, 0.2, 0.3], [0.15, 0.25, 0.35], [0.2, 0.3, 0.4]])
-B1 = np.array([[1, 1, 1], [1, 1, 1], [1, 1, 1]])
-c = 10
-calculated_mean1 = 1.5
-calculated_stddev1 = 0.91287092917528
+class TestImageUtils(unittest.TestCase):
+    """ Varon Ratio Tests """
```

Comment:
When I run `python -m unittest tests.test_image_utils`, I get "Ran 0 tests ..."

You probably want to put this in a class function called `test_varon_ratio(self)` and then replace the `assert` calls with `self.assertTrue()`.
For the `assert np.isclose()` calls, you can do `np.testing.assert_all_close()`, which takes in the same rtol value.